__tests__/topics.test.js

```diff
@@ -0,0 +1,56 @@
+const request = require('supertest')
+const app = require("../app")
+const seed = require("../db/seeds/seed")
+const db = require("../db/connection")
+const testData = require("../db/data/test-data/index")
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+    return db.end
+})
+
+describe("GET /api/healthcheck", () => {
+    test("200: responds with an okay message", () => {
+        return request(app)
+        .get("/api/healthcheck")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.msg).toBe("all okay");
+        });
+    });
+});
+
+describe("GET /api/topics tests", () => {
+    describe('Happy path tests', () => {
+        test('200: responds with an okay message', () => {
+            return request(app)
+            .get("/api/topics")
+            .expect(200)
+        })
+        test('responds with array', () => {
+            return request(app)
+            .get("/api/topics")
+            .then(( { body }) => {
+                const output = Array.isArray(body.topics)
+                const expected = true
+                expect(output).toEqual(expected)
+            })
+        })
+        test('responds with array of objects with properties of slug and description', () => {
+            return request(app)
+            .get("/api/topics")
+            .then(( { body }) => {
+                expect(body.topics).toHaveLength(3)
+                body.topics.forEach((topic) => {
+                    expect(topic).toMatchObject({
+                        slug: expect.any(String),
+                        description: expect.any(String)
+                    })
```

Comment:
This could all be one test, if we expect a (200) in the last test then all the cases you would want to test for would be included with one test